README.md

```diff
@@ -1 +1,70 @@
-# mv-math-magician
```

Comment:
- [ ] Please, add a readme file according to the microverse [template](https://github.com/microverseinc/readme-template). 🐶